src/107-Arduino-Servo-RP2040.cpp

```diff
@@ -66,6 +74,18 @@ void Servo::writeMicroseconds(uint16_t const pulse_width_us)
   pwm_set_chan_level(_slice_num, _channel, pulse_width_us);
 }
 
+void Servo::write(int angle)
+{
+  if (angle < 0)
```

Comment:
Well, this mapping is kind of problematic. Who is to say that all servos will always only have 180° range?

Reply:
Your point is indeed valid, and I share some of your doubts. However, according to the [Arduino Reference](https://www.arduino.cc/reference/en/libraries/servo/write/), it is stated that the range is "from 0 to 180". It appears more accurate to consider this as mapping the 0-180 range to the full extent of the swing, regardless of the actual angle.

On a different note, it seems that the [Arduino Pico](https://github.com/earlephilhower/arduino-pico/blob/c095fce5b2f26b03fa0e6374e3371a1799abb5b1/libraries/Servo/src/Servo.cpp#L124-L132) interprets a value of less than 200 as representing an angle.

What changes would you prefer?

Reply:
Personally I'd suggest to assume 360° and provide an overloaded constructor to define actual servo limits. The Arduino Pico approach is interesting because its more failsafe (i.e. more likely to obtain the desired result) for novice users but the libraries within this organisation target experienced users and therefore exactness and APIs semantically matching the expectation you have when calling them matter more.

Reply:
I see.

For now setting aside the special handling for Arduino-Pico, if we deviate from the rule that "in the `write` method, regardless of the actual angle, integers from 0 to 180 are mapped to a full swing," it seems to me that I cannot say that it is "providing methods with the Arduino API signature," which is the purpose of this PR. Well, I also think this is not a very good API though...

My view is that it would be better to leave `write` as it is and provide a separate function to specify the "actual" angle from 0 to 360 degrees if we need. Perhaps introducing a new `writeDegree` method and an additional overload for ctor or `attach` to specify the actual full swing angle might be the way to go.